Portfolio/src/assets/HomeView.scss

```diff
@@ -73,43 +75,43 @@
   }
 
   .home-page-right-container {
-    font-size: 2.5rem;
+    font-size: $font-size-small;
     align-items: flex-end;
   }
 }
 
 @media (min-width: 1281px) and (max-width: 1899px) {
   .home-page-left-container {
-    margin-top: 1rem;
+    margin-top: $margin-top-xsmall;
     align-items: start;
   }
 
   .home-page-title {
-    font-size: 70px;
+    font-size: $font-size-large;
   }
 
   .home-page-right-container {
-    font-size: 35px;
+    font-size: $font-size-small;
     align-items: flex-end;
   }
 }
 
 @media (min-width: 1900px) {
   .home-page-left-container {
-    margin-top: 1rem;
+    margin-top: $margin-top-xsmall;
     align-items: start;
   }
 
   .home-page-title {
-    font-size: 5rem;
+    font-size: $font-size-xlarge;
   }
   .home-page-right-container {
-    margin-top: 10rem; /*  $margin-top-large */
+    margin-top: $margin-top-large;
     font-size: 2.5rem;
```

Comment:
So my point arrises here. Regarding margin scaling. Here you have a margin of 2.5 rem. How will u name it since u already have xs, s, md, lg, xl? There is no space between s and md. So u already see the issues of naming variables in a way that does not scale


$font-size-xsmall: 1.5rem;
$font-size-small: 2rem;
$font-size-medium: 3rem;
$font-size-large: 4rem;
$font-size-xlarge: 5rem;

Reply:
Done